### ISSUE TITLE
Add Projects and Events to the Host Dashboard Reports Statistics

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#4726]
This is with regard to the following comment; https://github.com/opencollective/opencollective-frontend/pull/6873#discussion_r713649524. 

From @Betree :

>But what are the host's expectations when they visit this page? To me, it sounds like they would want to see all transactions happening under their fiscal hosting, regardless of the account types. The value proposition of this section is to show a clear view of how the money is flowing among your hosted initiatives; yet if you limit that to collectives you'll miss a great part of the transactions (some collectives now manage their budgets mainly through projects).
>
>I currently don't see any good reason to limit that to collectives. If it's a wording problem, then we should update the wording and say "Accounts" or "Hosted initiatives" rather than "Collectives".
>
>It would be nice to merge this PR, so let's not block on this but we should ask for design and iterate soon enough.

We are currently showing only collectives in the filters of this section as per the [following design](https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=13711%3A161925). As shown it seems that the page is using the word _Collectives_. I think we need to reword the wording of the page (esp the filter headings) to indicate the users that we are also showing Event and Project statistics. 

![Desktop](https://user-images.githubusercontent.com/12435965/134348466-588fdb3d-8b75-4a4a-801b-4911edd13ba2.png)
r

Simple change for the MVP is to use the word _Accounts_ or _Hosted Initiatives_ instead of _Collectives_ but I wonder if there's anything else need to be changed on this page. 🤔 

cc: @opencollective/design

**Comment by SudharakaP** [comment]
As per my chat with @Memo-Es I think we are moving towards adding data from events and projects but not including those events and projects in the filters. So the filters only show the collectives but when the collectives are chosen the stats from events and projects are reflected as well. Memo is working on surfacing events and projects more clearly on the UI so that the user knows that the stats include those as well. 

@Memo-Es : Hope I covered everything. Please feel free to comment if there's anything to add. 😉

**Comment by Betree** [comment]
> not including those events and projects in the filters

I don't get the reason for such a limitation. Let's say that OSC wants to see how money flowed for the various SustainOSS events, to compare the popularity of them across years. What's the rationale for not allowing them to do that?

**Comment by SudharakaP** [comment]
> > not including those events and projects in the filters
> 
> I don't get the reason for such a limitation. Let's say that OSC wants to see how money flowed for the various SustainOSS events, to compare the popularity of them across years. What's the rationale for not allowing them to do that?

Maybe @Memo-Es can explain this better, but I am quoting a discussion we had just so we are all in sync on the rational behind this; 

>I think we should display data from events and projects but not adding those events and projects as individual filters
>
>Some events have long names, a collective can have multiple projects, among other considerations that break the experience IMO
>I’m doing a round of copy-editing today and ask Alanna to have a look anyway
>
>I will also think on how to surface events and projects more clearly

Anyways, I am okay either way, so maybe let us wait for Alanna and Memo's feedback on this to make a decision. 😉

**Comment by SudharakaP** [comment]
Just to summarize what we discussed in today's meeting; 

1) For the MVP we will simply add support to filter by Collectives, Events and Projects; I've added this at https://github.com/opencollective/opencollective-frontend/pull/6873/commits/59d3a87e1a28b99a25b24450ded9d596f72f26e7 for the transactions view.

2) For the future; we have to discuss whether it is better to only let the user filter by collectives but show all the relevant the child collective stats. In this case we will need to change the wording (or some design changes) of the designs to give the user the indication the statistics include the child collectives as well.

**Comment by SudharakaP** [comment]
I don't think this issue is relevant anymore at least in the context of the current implementation. Since we have one centralized filter as per https://github.com/opencollective/opencollective-frontend/pull/7100 we already show collectives, events and projects there. The user is able to filter based on the projects and events. My original intention of opening this issue was because initially we only allowed users to filter based on collectives and then it was changed to allow uses to filter on projects, events etc (refer: https://github.com/opencollective/opencollective-frontend/pull/6873#discussion_r710835520). And this gave rise to some wording issues as we had paragraphs in the dashboard page which indicated we are showing stats of collectives. However those paragraphs are now removed and thus I suggest we close this issue and iterate in the future if there's any changes to this page is necessary. 🤔

**Comment by Betree** [comment]
@SudharakaP I think the underlying issue that hasn't been solved yet (unless I'm missing something) is that, as a host admin, when I pick "Babel" in the account filter, I would expect to see the financial data for Babel **and** all its events, projects.

@alanna you've been using this dashboard, is this the behavior that you would expect?

**Comment by alanna** [comment]
Yes we need these stats to include projects and events, otherwise the per-Collective view is not useful.

But I have completely stopped using these reports because of #5372 - the numbers aren't accurate and we can't rely on them.
